.changeset/afraid-llamas-know.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-test-data/core': patch
+'@commercetools-test-data/commons': patch
+---
+
+chore: update all deps
```

Comment:
I forgot to add the changeset for the PR #55 
I thought it should be fine to have it here.